gettingStarted.md

```diff
@@ -0,0 +1,354 @@
+
+Clients run anywhere, but their development requires further setup. Right now, clients should be developed in this repository or in a fork of it to make use of its setup.
+
+Then, reinstall the dependencies:
+```bash
+npm install
+```
+
+### 4.3 Using Plugins in Code – Detailed Explanation
+
+The provided `addPlugins.ts` file defines the `addPlugins` function, which integrates multiple plugins into the map application. Let’s break down the code and explain how each plugin is integrated and what configuration options are available.
+
+#### Overview of the `addPlugins` Function
+
+The `addPlugins` function expects a `core` object as a parameter. This `core` object represents the main instance of the POLAR application and provides methods to add plugins.
+
+```typescript
+export const addPlugins = (core) => {
+  setLayout(NineLayout);
+  // Adding the iconMenu plugin and other plugins
+  core.addPlugins([ /* List of Plugins */ ]);
+}
+```
+
+#### Configuration of the `iconMenu` Plugin
+
+The `iconMenu` plugin serves as a central menu for displaying and accessing other plugins. It is configured with a list of menu entries, each embedding another plugin.
+
+```typescript
+const iconMenu = IconMenu({
+  menus: [
+    {
+      plugin: LayerChooser({}), // Layer chooser plugin
+      icon: 'fa-layer-group',   // Icon for the plugin in the menu
+      id: 'layerChooser',
+    },
+    {
+      plugin: Draw({}),         // Drawing plugin
+      icon: 'fa-pencil',        // Icon for the plugin
+      id: 'draw',
+    },
+    {
+      plugin: Zoom({ renderType: 'iconMenu' }), // Zoom plugin
+      id: 'zoom',
+    },
+    {
+      plugin: Fullscreen({ renderType: 'iconMenu' }), // Fullscreen plugin
+      id: 'fullscreen',
+    },
+    {
+      plugin: GeoLocation({ renderType: 'iconMenu' }), // Geolocation plugin
+      id: 'geoLocation',
+    },
+    {
+      plugin: Attributions({
+        renderType: 'iconMenu',
+        listenToChanges: [
+          'plugin/zoom/zoomLevel',
+          'plugin/layerChooser/activeBackgroundId',
+          'plugin/layerChooser/activeMaskIds',
+        ],
+      }),
+      icon: 'fa-regular fa-copyright',
+      id: 'attributions',
+    },
+  ],
+  displayComponent: true,     // The IconMenu will be displayed on the map
+  initiallyOpen: 'layerChooser', // Specifies which menu is open by default
+  layoutTag: NineLayoutTag.TOP_RIGHT, // Position of the iconMenu on the map
+});
+```
+
+#### Adding Plugins to the Map
+
+The `core.addPlugins` method is used to add the list of plugins to the map application. Here’s a detailed look at some of these plugins:
+
+```typescript
+core.addPlugins([
+  AddressSearch(
+    merge({}, defaultOptions, {
+      layoutTag: NineLayoutTag.TOP_LEFT, // Position of the plugin on the map
```

Comment:
This comment gets really repetitive. Is it necessary here?

Reply:
I guess the chapter explaining the layout is enough to do it without the comments.

Reply:
done